mlflow/store/artifact/artifact_repo.py

```diff
@@ -150,6 +150,15 @@ def _download_file(self, remote_file_path, local_path):
         """
         pass
 
+    def delete_artifacts(self, artifact_path):
```

Comment:
We can probably make `artifact_path` an optional argument; if unspecified, all artifacts *under* the root are deleted. Can you make this change and specify the behavior for `artifact_path=None` in the method docs?

Reply:
Done